### PR TITLE
feature(users): add 'role' attribute

### PR DESCRIPTION
**Pull request by ruslanshakirov**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,5 @@ class User < ActiveRecord::Base
     :recoverable, :rememberable, :trackable, :validatable
 
   validates :full_name, presence: true
+  enum role: %i(user administrator)
```
Comment (vladimirbazhanov) [inline app/models/user.rb]: can you pls user string here and keep words 'user' or 'administrator' here?

Comment (ruslanshakirov) [inline app/models/user.rb]: @vladimirbazhanov what do you mean?
Rubocop will give me offenses if I don't use word array.

Comment (ruslanshakirov) [inline app/models/user.rb]: @vladimirbazhanov something like that?
enum role: { user: "user", administrator: "administrator" }

Comment (vladimirbazhanov) [inline app/models/user.rb]: @ruslanshakirov yes, this will be much better

File: db/migrate/20160811130642_add_role_to_users.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :role, :integer, default: 0
```
Comment (vladimirbazhanov) [inline db/migrate/20160811130642_add_role_to_users.rb]: and pls create default: 'user' thanks!

**Pull request merged**